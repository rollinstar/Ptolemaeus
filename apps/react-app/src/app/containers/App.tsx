import React from 'react';
import styled from 'styled-components';

import { Button } from 'react-ui';

const AppPage = styled.div`
    font-size: 15px;
    color: 1a202c;
`;

export const App = () => {
    return (
        <AppPage>
            <h1>Ptolemaeus</h1>
            <Button variant='outlined' onClick={() => console.log('Button')}>
                상세보기
            </Button>
        </AppPage>
    );
};
