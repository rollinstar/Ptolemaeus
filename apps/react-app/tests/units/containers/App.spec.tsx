import React from 'react';
import { App } from '../../../src/app/containers/App';
import { render } from '../../test-util';
import 'jest-styled-components';

describe('<App /> test', () => {
    it('matches snapshot', () => {
        const app = render(<App />);
        expect(app).toMatchSnapshot();
    });
});
