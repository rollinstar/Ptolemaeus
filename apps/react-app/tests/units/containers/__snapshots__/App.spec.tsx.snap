// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> test matches snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": .c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: transparent;
  border-radius: 4px;
  border: 0;
  margin: 0;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  font-size: 0.8rem;
  font-weight: 500;
  line-height: 1.7;
  -webkit-letter-spacing: 0.02em;
  -moz-letter-spacing: 0.02em;
  -ms-letter-spacing: 0.02em;
  letter-spacing: 0.02em;
  min-width: 65px;
  padding: 6px 8px;
  color: var(--primary-color);
  -webkit-transition: background-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, box-shadow 250ms cubic-bezier(0.4,0,0.2,1) 0ms,border-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, color 250ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, box-shadow 250ms cubic-bezier(0.4,0,0.2,1) 0ms,border-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, color 250ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c1.contained {
  background-color: var(--primary-color);
  color: #edf2f7;
  box-shadow: rgb(0 0 0 / 20%) 0px 3px 1px -2px,rgb(0 0 0 / 14%) 0px 2px 2px 0px, rgb(0 0 0 / 12%) 0px 1px 5px 0px;
}

.c1.outlined {
  border: 1px var(--primary-color) solid;
  box-shadow: rgb(0 0 0 / 20%) 0px 3px 1px -2px,rgb(0 0 0 / 14%) 0px 2px 2px 0px, rgb(0 0 0 / 12%) 0px 1px 5px 0px;
}

.c1.small {
  font-size: 0.6rem;
}

.c1.large {
  font-size: 0.9rem;
}

.c1:disabled {
  background-color: var(--disabled-color);
}

.c0 {
  font-size: 15px;
  color: 1a202c;
}

<body>
    <div>
      <div>
        <div
          class="c0"
        >
          <h1>
            Ptolemaeus
          </h1>
          <button
            class="c1 outlined"
          >
            상세보기
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": .c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: transparent;
  border-radius: 4px;
  border: 0;
  margin: 0;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  font-size: 0.8rem;
  font-weight: 500;
  line-height: 1.7;
  -webkit-letter-spacing: 0.02em;
  -moz-letter-spacing: 0.02em;
  -ms-letter-spacing: 0.02em;
  letter-spacing: 0.02em;
  min-width: 65px;
  padding: 6px 8px;
  color: var(--primary-color);
  -webkit-transition: background-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, box-shadow 250ms cubic-bezier(0.4,0,0.2,1) 0ms,border-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, color 250ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, box-shadow 250ms cubic-bezier(0.4,0,0.2,1) 0ms,border-color 250ms cubic-bezier(0.4,0,0.2,1) 0ms, color 250ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c1.contained {
  background-color: var(--primary-color);
  color: #edf2f7;
  box-shadow: rgb(0 0 0 / 20%) 0px 3px 1px -2px,rgb(0 0 0 / 14%) 0px 2px 2px 0px, rgb(0 0 0 / 12%) 0px 1px 5px 0px;
}

.c1.outlined {
  border: 1px var(--primary-color) solid;
  box-shadow: rgb(0 0 0 / 20%) 0px 3px 1px -2px,rgb(0 0 0 / 14%) 0px 2px 2px 0px, rgb(0 0 0 / 12%) 0px 1px 5px 0px;
}

.c1.small {
  font-size: 0.6rem;
}

.c1.large {
  font-size: 0.9rem;
}

.c1:disabled {
  background-color: var(--disabled-color);
}

.c0 {
  font-size: 15px;
  color: 1a202c;
}

<div>
    <div>
      <div
        class="c0"
      >
        <h1>
          Ptolemaeus
        </h1>
        <button
          class="c1 outlined"
        >
          상세보기
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
